contracts/ronin/profile/Profile.sol

```diff
@@ -448,6 +511,28 @@ contract Profile is IProfile, ProfileXComponents, Initializable {
     _setVRFKeyHash(_profile, vrfKeyHash);
   }
 
+  /**
+   * @inheritdoc IProfile
+   */
+  function changeSequencerAddr(address id, address sequencer) external onlyAdmin {
+    CandidateProfile storage _profile = _getId2ProfileHelper(id);
+
+    _requireNonZeroAndNonDuplicated(RoleAccess.SEQUENCER, sequencer);
+    _requireRollupCreated(_profile);
+    _setSequencer(_profile, sequencer);
```

Comment:
As this function required the sequencer address to not be registered before by calling `_requireNonZeroAndNonDuplicated`, we must add the registered sequencer address to _registry for tracking registered address.

Reply:
fixed.